packages/devtools-components/src/tree.css

```diff
@@ -33,17 +34,11 @@
 
 .tree-indent {
   display: inline-block;
-  width: 12px;
-  margin-inline-start: 5px;
+  width: 16px;
   border-inline-start: 1px solid #A2D1FF;
   flex-shrink: 0;
 }
 
-/* Align with expandables siblings (where we have the arrow) */
-.tree-node[data-expandable="false"] .tree-indent:last-of-type {
```

Comment:
Ran some experiments and you're right. Storage and Accessibility sets their own `tree-indent`. Console uses `tree.css`'s `.tree-indent`. 


<details><summary><b>IMG: Storage and Accessibility in Firefox build with this PR</b></summary>
<p>I set `.tree-indent { ... }` to `background-color: lime !important` to see if it would show in these panels, and it didn't.</p>
<img src="https://user-images.githubusercontent.com/15959269/53890548-69bcdb00-3ff6-11e9-9181-6e27aef3313d.png">
<img src="https://user-images.githubusercontent.com/15959269/53890549-6a557180-3ff6-11e9-9b00-4566afda8782.png">
</details><br>

Taking out `:last-of-type` only affects Debugger and Console, and they're both aligned properly as expected after taking it out, so we should be okay.

<details><summary><b>IMG: Tree in Console and Debugger(Sources/Scopes)</b></summary>
<p>I set `.tree-indent { ... }` to `background-color: lime !important`.</p>
<h4>Console</h4>
<img src="https://user-images.githubusercontent.com/15959269/53890178-b5bb5000-3ff5-11e9-9809-6020d7949a34.png">
<h4>Debugger (Sources)</h4>
<img src="https://user-images.githubusercontent.com/15959269/53890247-d5527880-3ff5-11e9-975f-898c3d61908e.png">
<h4>Debugger (Scopes)</h4>
<img src="https://user-images.githubusercontent.com/15959269/53890279-e4d1c180-3ff5-11e9-8cd2-ba17f4ac873d.png">
</details>

Reply:
Well, now we know why `.tree-indent` had a `margin-inline-start`: it centers the border (see Console) under the arrow. So we may need to keep that. :/

Reply:
@nchevobbe What do you think, do we expect to have this border style only in Console? Should we define it in webconsole.css?

Alternatively, I would love for this style to be opt-in, using an optional class, e.g.:

```html
<span class="tree-indent tree-indent-border">
```
vs:

```html
<span class="tree-indent">
```